man/systemd-sysusers.xml

```diff
@@ -94,6 +94,12 @@
         paths. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--direct</option></term>
```

Comment:
hmm, i am a bit afraid of getopt()'s parsing interfering with the arguments following. Hence I think we should document this as `<option>--direct</option> --` maybe? i.e. clarify that this should be followed by `--`, to disable getopt()'s interfering?

Reply:
Valid configuration lines start with a letter or "#", so no interference is possible in any realistic use.

Reply:
I mean in the spec macro I indeed use `--` just to be safe, but I don't think it's necessary to document that.

Reply:
I think we should still avoid any potential confusion there and just use a switch with an argument...